docs/variants/novacustom_ns5x/releases.md

```diff
@@ -14,6 +14,45 @@ For details about our release process please read
 Test results for this platform can be found
 [here](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit?usp=sharing).
 
+## v1.2.0 - 2022-05-26
+
+Test results for this release can be found
+[here](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=1695997523).
+
+### Added
+
+- Persistent RGB keyboard settings
+- Increased power limits to CPU defaults (28W PL1 / 35W PL2)
+
+### Fixed
+
+- CVE-2022-29264 SMM loader vulnerability
```

Comment:
We can link to e.g. https://nvd.nist.gov/vuln/detail/CVE-2022-29264 , can we?

Reply:
Added